### ISSUE TITLE
Convolutions' padding - student/teacher model

### ISSUE THREAD

**Issue by lorenzolat1** [nelson1425/EfficientAD#4]
Hi! I want to congratulate with you for the implementation of EfficientAD, thank you for your work. I would like to better understand some parts of your implementations, specifically the student/teacher model. In the paper the model has a fixed value for the padding in the convolutions resulting in an output with the shape 384x64x64, instead in your implementation the padding is set by the boolean's variable and the output's shape might be different (384x56x56). Is there a reason behind it?

**Comment by nelson1425** [comment]
Yes it is to follow the paper where padding can be disabled to speed up the inference.

**Comment by kywish** [comment]
> Yes it is to follow the paper where padding can be disabled to speed up the inference.
Why pad 4 pixels before resizing instead of resizing directly。
[map_combined = torch.nn.functional.pad(map_combined, (4, 4, 4, 4))](https://github.com/nelson1425/EfficientAD/blob/ce0275cbe1be590a5dbf18959db430ffdaf98561/efficientad.py#L279)
